docs/NEW_POSTS_NOTIFICATION_FEATURE.md

```
@@ -1,113 +1,158 @@
 
-*   **`HomePage` -> `NewPostService`** :
-    *   S'abonne à `getNewPostsCount()` pour l'affichage du badge.
-    *   Appelle `getNewPosts()` pour récupérer la liste à afficher dans le modal.
-    *   Appelle `markPostAsSeen()` après qu'un post a été consulté.
+async initStorage() {
+  await this.storage.create(); // Initialise le driver de stockage
+  const seenPosts = await this.storage.get(SEEN_POSTS_KEY);
+  this.seenPosts.next(seenPosts || []);
+}
+```
+-   Le service utilise `@ionic/storage-angular` pour récupérer la liste des identifiants (`ID`) des posts que l'utilisateur a déjà "vus" lors de sessions précédentes. Cette liste est stockée sous la clé `SEEN_POSTS_KEY`.
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)